web/src/drafts.ts

```diff
@@ -580,6 +580,7 @@ export type FormattedDraft =
           is_stream: false;
           draft_id: string;
           recipients: string;
+          no_recipients: boolean;
```

Comment:
I'd choose a name for this that isn't negative ... maybe "has_recipient_data" or "recipient_data_set"?